[Dynamic Yield] adding instructions for ip allowlisting (#7017)
segmentio/segment-docs: src/connections/destinations/catalog/actions-dynamic-yield-audiences/index.md

**Dynamic Yield by Mastercard Audiences** is an Audience Destination which must be first connected to an Engage Space before it can be connected to individual Engage Audiences. The steps below outline how to connect the Destination to an Engage Space and then to an Audience.

### Enable IP allowlisting

> Dynamic Yield requires that data sent by Segment originate from a fixed IP range. As a prerequisite to using the **Dynamic Yield by Mastercard Audiences** Destination, Segment Business Tier customers must enable the **IP Allowlisting** feature on their Workspace. This feature is not available for non Segment Business Tier customers. 

Reviewer: ```suggestion
Dynamic Yield requires that data sent by Segment originate from a fixed IP range. As a prerequisite to using the **Dynamic Yield by Mastercard Audiences** Destination, Segment Business Tier customers must enable the **IP Allowlisting** feature on their workspace. This feature is only available for Segment Business Tier customers. 
```

> 1. From your Segment Workspace, navigate to **Settings > Workspace Settings > Destination IP Settings**
> 2. Click **Enable IP Whitelisting** to enable the feature. 

Reviewer: ```suggestion
To enable IP allowlisting:
1. Navigate to **Settings > Workspace Settings > Destination IP Settings** in your Segment workspace.
2. Click **Enable IP allowlisting** to enable the feature. 
```
